nixos/modules/system/boot/loader/limine/limine.nix

```
@@ -47,6 +50,40 @@ in
   options.boot.loader.limine = {
     enable = lib.mkEnableOption "the Limine Bootloader";
 
+    enableSecureBoot = lib.mkEnableOption null // {
+      description = ''
+        Whether to use sign the limine binary with sbctl.
+        This requires you to already have generated the keys and enrolled them with {command}`sbctl`.
+
+        To create keys use {command}`sbctl create-keys`.
+
+        To enroll them first reset secure boot to "Setup Mode". This is device specific.
+        Then enroll them using {command}`sbctl enroll-keys -m`.
+
+        You can now rebuild your system with this option enabled.
+
+        Afterwards turn setup mode off and enable secure boot.
```

```suggestion
        Whether to sign the limine binary with sbctl for secure boot setups.
        
        This requires you to already have generated the keys and enrolled them with {command}`sbctl`.
        
        To create keys use {command}`sbctl create-keys`.

        To enroll them first reset secure boot to "Setup Mode". This is device specific.
        Then enroll them using {command}`sbctl enroll-keys -m`.

        You can now rebuild your system with this option enabled.

        Afterwards turn setup mode off and enable secure boot.
```

AFAIK I did the full command with:
```
 sudo sbctl enroll-keys --microsoft --firmware-builtin --tpm-eventlog
```
 
 when setting up lanzaboote, but I suppose this is enough.